docs/source/ecosystem/index.rst

```diff
@@ -0,0 +1,116 @@
+:orphan:
+
+.. include:: /include/substitution.rst
+
+.. _applications:
+
+*********************
+PeekingDuck Ecosystem
+*********************
+
+This sections covers the extensions to the PeekingDuck ecosystem.
+
+
+.. _pkd_viewer:
+
+PeekingDuck Viewer
+==================
+
+The PeekingDuck Viewer offers you an interactive GUI application to manage and run
+PeekingDuck pipelines, and to view and analyze the output video.
+
+
+Running the Viewer
+------------------
+
+The PeekingDuck Viewer can be activated using the CLI ``--viewer`` option:
```

Comment:
I believe this needs to be used in conjunction with `output.screen` node else the window looks empty. This is quite important and should be mentioned early with emphasis.

Should we go one step further and even throw an error if `output.screen` is not used?

Reply:
There is another option: if `--viewer` is activated, always render the output, regardless if `output.screen` node is present or not. The way to do this is to capture the `img` from the data pool at the end of the current pipeline iteration and render it to the viewer window.